README.md

```diff
@@ -1,19 +1,48 @@
-## Features
-
-- signup
-    - With google account
-    - or local account
-- User can add multiple services for email
-    - google
-    - outlook
-    - yahoo
-    - custom email (requires smtp)
-- User can select primary service for sending emails
-- All the emails from all services will be visible at one single place
-- While sending email user can select from which service he wants to send email
-    - primary email can be used by default (if selected)
-- User can set reminders and todos
-
-## Future plans
-
-- Create mobile with same features
+# Technologies Used
+
+- React
+- Redux
+- Tailwind CSS
+
+# Features
+
+- Admin dashboard
+- User settings and or user dashboard
+- send emails
+- recive emails
+- Connections through other parties ie discord, twitter, facebook etc..
+
+# Setup
+
+## Windows
+
+```
+git clone https://github.com/MultiEmail/MultiEmail-backend.git
+cd MultiEmail-backend
+yarn install
+yarn dev
+```
+
+# Linux
+
+```
+git clone https://github.com/MultiEmail/MultiEmail-backend.git && cd MultiEmail-backend && yarn install && yarn dev
```

Comment:
Also this “ git clone https://github.com/MultiEmail/MultiEmail-frontend.git && cd MultiEmail-frontend && yarn install && yarn dev